client.go

```diff
@@ -491,7 +481,7 @@ func (c *Client) sendHub() {
 				// And get rid of the timeout workaround.
 				c.LocalKite.Log.Error("error sending %q: %s", msg, err)
 
-				if err == sockjsclient.ErrSessionClosed {
+				if err == sockjsclient.ErrSessionClosed || strings.Contains(err.Error(), errClosing) {
```

Comment:
I agree, the string matching here is awful, but it's here to catch the cases when the connection was closed by the server - the closeChan is only closed explicitly client-side.

Ideally if I were to refactor this further I would just remove closeChan and close the session in Close method - closed session should interrupt all Read / Writes with ErrSessionClosed (it's done already for xhr and partially for websocket) and should translate http.errClosing to ErrSessionClosed as well (not done). The less ways there's to change connection state the better.